core/modules/system/system.module

```diff
@@ -837,16 +837,16 @@ function system_menu() {
     'access arguments' => array('access administration pages'),
     'file' => 'system.admin.inc',
   );
-  $items['admin/config/search/clean-urls'] = array(
-    'title' => 'Clean URLs',
-    'description' => 'Enable or disable clean URLs for your site.',
+  $items['admin/config/search/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Handling URL-related configuration of your site.',
     'page callback' => 'backdrop_get_form',
-    'page arguments' => array('system_clean_url_settings'),
+    'page arguments' => array('system_urls_settings'),
     'access arguments' => array('administer site configuration'),
     'file' => 'system.admin.inc',
     'weight' => 5,
   );
```

Comment:
You mean something like (add to `function system_menu()`):
```php
  $items['admin/config/search/clean-urls'] = array(
    'page callback' => 'system_urls_callback',
    'access arguments' => array('access administration pages'),
    'type' => MENU_CALLBACK,
    'file' => 'system.admin.inc',
  );
```
and add to file `system.admin.inc`:
```php
/**
 * Menu backwards-compatible item callback
 * @deprecated since version 2.0
 */
function system_urls_callback() {
  backdrop_goto('admin/config/search/urls');
}
```
? 
Works for me.